src/bindings/node/node_osrm.cpp

```diff
@@ -0,0 +1,507 @@
+#include "osrm/engine_config.hpp"
```

Comment:
Yeah but you can also have `src/python` if you want that. I don't think we will have namespace conflicts 😉